### DISCUSSION TITLE
basis points unit (bps) support in grafana

### DISCUSSION THREAD

**Discussion by sanderp42** [grafana/grafana#52511] [Dashboards, Panels & Visualizations]
We use grafana for our crypto trading operations. Displaying margins is typically done in basis points (bps) as margins are in the order of 0.03-0.001%, however currently only percent (%) is supported. 1 bps = 0.01% 

This is a very easy fix which is a nice and small improvement. 

So then you would have standard **options -> unit -> mics -> basis points**

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
